com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Utility/HDUtils.cs

```diff
@@ -642,7 +642,8 @@ public void ComputePackedMipChainInfo(Vector2Int viewportSize)
                 }
                 while ((mipSize.x > 1) || (mipSize.y > 1));
 
-                textureSize = new Vector2Int((int)((float)hardwareTextureSize.x * textureScale.x), (int)((float)hardwareTextureSize.y * textureScale.y));
+                textureSize = new Vector2Int(
```

Comment:
IIRC there is a disparity of rounding behaviour between HW and SW, have you checked both with a collection of resolution? (odd resolutions will be the problem I think). 

If you didnt I guess that's a ping for QA

Reply:
It should work since that computation is what writes on acualWidth and actualHeight.
Will indeed ask QA to test for this.